mimic/test/test_elasticsearch.py

```diff
@@ -0,0 +1,230 @@
+import mock
+from mock import create_autospec
+
+from twisted.internet.task import Clock
+from twisted.trial.unittest import SynchronousTestCase
+from twisted.web import http
+
+from mimic.core import MimicCore
+from mimic.file_based_responses import elasticsearch
+from mimic.resource import MimicRoot
+from mimic.rest import elasticsearch_api
+from mimic.test.helpers import json_request
+
+
+class ElasticSearchAPITests(SynchronousTestCase):
+
+    """
+    Tests for the ElasticSearch API
+    """
+
+    # JSON data to be used by mocked JSON load function
+    unittest_json = [
```

Comment:
As mentioned in earlier comment, this should be eliminated.  Either test the actual JSON as implemented in the service, or better yet, eliminate the canned JSON and synthesize a response based on some elasticsearch model objects.